housekeeping-vacuum-tool/README.md

```diff
@@ -60,11 +60,11 @@ The `dry-run` option allows you to observe the status of paths on the file syste
 |`tables.table-name`|Yes| The Hive table name for the table the vacuum tool will interrogate.|
 |`housekeeping.schema-name`|No|The schema name that is used in your housekeeping instance. Defaults to `housekeeping`.|
 |`housekeeping.h2.home`|No|The path to your H2 filesystem database specified as a full path. Defaults to `$HOME/data`.|
```

Comment:
I wasn't sure about this, but I got the feeling that this row should be

|`housekeeping.h2.database`|No|The path to your H2 filesystem database specified as a full path. Defaults to `${instance.home}/data/${instance.name}/${housekeeping.schema-name}` (where `instance.home`, `instance.name` and `housekeeping.schema-name` can be configured separately for more fine-grained control).|
| :-- | :-- | :-- |

Reply:
I think you are right from the code: `.put("housekeeping.h2.database", "${instance.home}/data/${instance.name}/housekeeping")`